Modules/CalcPerform.lua

```diff
@@ -14,9 +14,308 @@ local m_ceil = math.ceil
+				if breakdown then
+					breakdown.SourceTriggerRate = {
+						s_format("%.2f ^8(%s attacks per second)", sourceAPS, source.activeEffect.grantedEffect.name),
+						s_format("/ %.2f ^8(simulated impact of linked spells)", sourceAPS / output.SourceTriggerRate),
+						s_format("= %.2f ^8per second", output.SourceTriggerRate),
+						s_format(""),
+						s_format("Simulation Breakdown"),
+						s_format("Simulation Duration: %.2f", simBreakdown.simTime),
+					}
+					breakdown.SimData = {
+						rowList = { },
+						colList = {
+							{ label = "Rate", key = "rate" },
+							{ label = "Skill Name", key = "skillName" },
+							{ label = "Slot Name", key = "slotName" },
+							{ label = "Gem Index", key = "gemIndex" },
+						},
+					}
+					for _, rateData in ipairs(simBreakdown.rates) do
+						local t = { }
+						for str in string.gmatch(rateData.name, "([^_]+)") do
+							table.insert(t, str)
+						end
+
+						local row = {
+							rate = rateData.rate,
+							skillName = t[1],
+							slotName = t[2],
+							gemIndex = t[3],
+						}
+						t_insert(breakdown.SimData.rowList, row)
+					end
+				end
+			else
+				if breakdown then
+					breakdown.SourceTriggerRate = {
+						s_format("%.2f ^8(%s attacks per second)", sourceAPS, source.activeEffect.grantedEffect.name),
+						s_format("/ %.2f ^8(number of linked active spells to trigger)", skillRotationImpact),
+						s_format("= %.2f ^8per second", output.SourceTriggerRate),
+					}
+				end
+			end
+		end
+		-- Set trigger rate
+		trigRate = m_min(output.SourceTriggerRate, output.ActionTriggerRate)
+	else
+		output.SourceTriggerRate = output.ActionTriggerRate / skillRotationImpact
+		if breakdown then
+			breakdown.SourceTriggerRate = {
+				s_format("%.2f ^8(%s triggers per second)", output.ActionTriggerRate, source.activeEffect.grantedEffect.name),
+				s_format("/ %.2f ^8(number of linked active spells to trigger)", skillRotationImpact),
+				s_format("= %.2f ^8per second", output.SourceTriggerRate),
+			}
+		end
+		-- Set trigger rate
+		trigRate = output.SourceTriggerRate
+	end
+
+	-- Adjust for server tick rate
+	local trigCD = 1 / trigRate
+	if breakdown then
+		breakdown.ServerTriggerRate = {
+			s_format("%.2f ^8(smaller of 'cap' and 'skill' trigger rates)", trigRate),
+		}
+	end
+	output.ServerTriggerRate = trigRate
+	return trigRate
+end
+
+-- Account for APS modifications do to Dual Wield or Simultaneous Strikes
+local function calcDualWieldImpact(env, sourceAPS, sourceName)
+	-- See if we are dual wielding
+	local dualWield = false
+	if env.player.weaponData1.type and env.player.weaponData2.type then
+		dualWield = true
+		sourceAPS = sourceAPS * 0.5
+	end
+
+	-- See if we are using a trigger skill that hits with both weapons simultaneously
+	if dualWield and (sourceName == "Cleave" or sourceName == "Dual Strike" or sourceName == "Viper Strike" or sourceName == "Riposte") then
```

Comment:
Let's use the `skill_double_hits_when_dual_wielding` mod/flag instead for future proofing, here

Reply:
Ooo... nice. Didn't realize we had this. Awesome. Fixed.